ONSdigital/dp-search-data-extractor
# Schema update for an array of strings

### What

* Schema change for an array of strings

### How to review

* check if code make sense
* build, audit, unit works fine

### Who can review

* anyone not me
